bedrock/products/views.py

```diff
@@ -516,3 +526,14 @@ def monitor_landing_page(request):
     context = {"breach_scan_url": breach_scan_url}
 
     return l10n_utils.render(request, template_name, context, ftl_files=ftl_files)
+
+
+@require_safe
+def vpn_resource_center_redirect(request, slug, old_template, ftl_files):
+    locale = l10n_utils.get_locale(request)
+    slug = "what-is-an-ip-address"
+    redirect_link = f"/{locale}/products/vpn/resource-center/{slug}/"
+    if active_locale_available(slug, locale):
+        return redirect(redirect_link)
+    else:
+        return l10n_utils.render(request, template=old_template, ftl_files=ftl_files)
```

Comment:
In cases where neither template is translated we are currently displaying the old template at the old URL in English at their locale. I think in these cases we usually forward them to the canonical English locale. 

Example: http://localhost:8000/ach/products/vpn/more/what-is-a-vpn/

Reply:
I don't think this is a result of the changes Pmac and I merged as reverting it didn't fix it locally but I didn't spend too long looking into it.